test/support/api/flows.js

```diff
@@ -1,5 +1,55 @@
 import _ from 'underscore';
-import { getResource, getRelationship } from 'helpers/json-api';
+import { getResource, getIncluded, getRelationship } from 'helpers/json-api';
+
+Cypress.Commands.add('routePatientFlows', (mutator = _.identity, patientId) => {
```

Comment:
it probably does make sense to create some sort of data generator function that these two can share at some point as you were moving towards earlier.. but we can circle back around.